src/app/admin/ownership/ownership.component.html

```diff
@@ -144,4 +144,161 @@ <h3>{{'COMING_SOON' | translate}}</h3>
        
 <!-- 
     </div> -->
-</div>
+<!-- </div> -->
+
+<div class="col-md-12 pr-4 mb-6">
+    <div class="step p-5">
+        <div class="mb-2">
+            <span class="fs-16 fw-bold mb-2">{{'HOW_CAN_ENTITIES_LOGIN' | translate}}</span>
+        </div>
+        <div class="card w-100 p-3">
+            <div class="px-3">
+                <div class="mb-3">
+                    <span class="fs-12 fw-bold">{{'UNIQUE_FIELD' | translate}}</span><br />
+                    <select name="uniqueField" id="uniqueField"
+                        class="w-50 br-4 border-1 border-light fs-14 p-3">
+                        <option value="option1">option1</option>
+                        <option value="option2">option2</option>
+                    </select>
+                </div>
+                
+                <div class="mb-3">
+                    <div class="row">
+                        <div class="col-3">
+                            <span class="fs-12 fw-bold">{{'AUTHENTICATION_TYPE' | translate}}</span><br />
+                            <select name="authType" id="authType"
+                                class="w-100 br-4 border-1 border-light fs-14 p-3">
+                                <option value="" disabled selected hidden>{{'SELECT_FIELD' | translate}}</option>
+                                <option value="username">{{'OTP' | translate}}</option>
+                            </select>
+                        </div>
+
+                        <div class="col-4">
+                            <span class="fs-12 fw-bold"></span><br />
```

Comment:
@Prasang023 is <span> tag is required here? if not remove it

Reply:
It is required to align all the columns in the row as first column is having the label in ```span``` tag while other two don't have hence span tag is empty. I'll remove the classes in the tag, they're of no use.